RecoParticleFlow/PFClusterProducer/src/PFClusterEMEnergyCorrector.cc

```diff
@@ -237,6 +235,21 @@ void PFClusterEMEnergyCorrector::correctEnergies(const edm::Event &evt,
     return;
   }
 
+  const double meanlimlow = -0.336;
+  const double meanlimhigh = 0.916;
+  const double meanoffset = meanlimlow + 0.5*(meanlimhigh-meanlimlow);
+  const double meanscale = 0.5*(meanlimhigh-meanlimlow);
+  
+  const double sigmalimlowEB = 0.001;
+  const double sigmalimhighEB = 0.4;
+  const double sigmaoffsetEB = sigmalimlowEB + 0.5*(sigmalimhighEB-sigmalimlowEB);
+  const double sigmascaleEB = 0.5*(sigmalimhighEB-sigmalimlowEB);  
+
+  const double sigmalimlowEE = 0.001;
+  const double sigmalimhighEE = 0.1;
```

Comment:
except for this magic number, everything else appears to be still identical.
Please reduce the copy-paste.
[the first 4 lines starting from meanlimlow are certainly the same]